addons/website_blog/static/src/js/website_blog.editor.js

```diff
@@ -335,6 +335,17 @@ options.registry.BlogPostTagSelection = options.Class.extend({
         }
         return this._super(...arguments);
     },
+    /**
+     * @override
+     */
+    async _computeWidgetState(methodName, params) {
+        if (methodName === 'addTag') {
+            // Even if the value will still be empty we need to return something here
+            // because we want the select to be filled by the default value of setValue() wich is "/".
+            return null;
+        }
+        return this._super(...arguments);
+    },
```

Comment:
@qsm-odoo Im not 100% sure about this solution but I think the use of the we-select is really unique here ...

Reply:
This is the right fix :+1: I'd changed the message to not mention code implementation through, something like:
```
// The related widget allows to select a value but then resets its state to a non-selected value
```
and instead of `null`, returns an empty string (as the method is supposed to return a string)

Reply:
@qsm-odoo Done